### PR TITLE
Add error prone and google java format plugins as project requirements

### PR DESCRIPTION
**Pull request by patflynn**
@etanshaul what do you think about this?

### CODE REVIEW

**Comment by etanshaul** [comment]
what actually happens? It forces you to install the plugins?

File: .idea/externalDependencies.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ExternalDependencies">
+    <plugin id="Error-prone plugin" />
+    <plugin id="google-java-format" />
+  </component>
+</project>
```
Comment (etanshaul) [inline .idea/externalDependencies.xml]: maybe newline here?

Comment (patflynn) [reply .idea/externalDependencies.xml]: IJ generated file when using the UI. gonna leave as is as I think it'll get reformatted any time it's changed from settings.

Comment (etanshaul) [reply .idea/externalDependencies.xml]: sounds good

**Pull request merged**